src/main/java/org/jenkinsci/plugins/parallel_test_executor/ParallelTestExecutor.java

```diff
@@ -61,6 +64,10 @@ public String getTestJob() {
     public String getPatternFile() {
         return patternFile;
     }
+    
+    public boolean getNoJava() {
```

Comment:
Bean getters for `boolean` conventionally use `is` rather than `get`.

Also I would recommend inverting the sense so that the visible property is called `java`, though the field could keep the current sense (`noJava`) to make it easier to retain serialization compatibility.